app/logical/scraper/trello.rb

```diff
@@ -0,0 +1,61 @@
+module Scraper
+  # https://developer.atlassian.com/cloud/trello/rest
+  class Trello < Base
+    STATE = :before
+    OPTIONAL_CONFIG_KEYS = %i[trello_key trello_token].freeze
+
+    def initialize(artist_url)
+      super
+      @before = nil
+    end
+
+    def fetch_next_batch
+      url = "https://api.trello.com/1/boards/#{api_identifier}/cards"
+      json = fetch_json(url,
+        params: {
+          attachments: true,
+          attachment_fields: "id,date,url",
+          fields: "id,desc,shortLink,name",
+          limit: 1000,
+          sort: "-id",
+        }.tap { |h| h[:before] = @before if @before }
+         .tap { |h| h[:key] = Config.trello_key if Config.trello_key }
+         .tap { |h| h[:token] = Config.trello_token if Config.trello_token },
```

Comment:
It doesn't look like it particularly cares about empty request parameters

```suggestion
          before: @before,
          key: Config.trello_key,
          token: Config.trello_token,
        },
```